demo/index.html

```diff
@@ -322,7 +293,13 @@ <h3>tangy-form with dynamically-populated checkboxes</h3>
                         required>
                 </tangy-input>
 
-                <tangy-checkboxes name="curriculum" label="Curriculum">
+                <tangy-checkboxes
+                        name="curriculum"
+                        label="Curriculum"
+                        options-list-source="forms.json"
+                        options-list-excludes="class-registration,student-registration"
+                        options-list-properties="id,title"
+                        options-list-exclude-by="id">
```

Comment:
Demo is great for examples. Would love to see a test. See how we test `<tangy-location>` [here](https://github.com/Tangerine-Community/tangy-form/blob/master/test/tangy-location_test.html#L48). Note the use of `done()` to finish an async test and also the `'location-list-loaded'` event that TangyLocation fires in order to make it testable.

Reply:
We now have a test in my latest commit.